### ISSUE TITLE
x86_64-pc-windows-gnu: version `GLIBC_2.28' not found

### ISSUE THREAD

**Issue by bnheise** [cross-rs/cross#1205]
### Checklist

- [X] I've looked through the [issues and pull requests](https://github.com/cross-rs/cross/issues?q=) for similar reports

### Describe your issue

This may not be a bug, but rather my lack of understanding of cross compiling to windows.

I'm trying to cross compile for x86_64-pc-windows-gnu but I consistently get this error message:

```
Status: Downloaded newer image for ghcr.io/cross-rs/x86_64-pc-windows-gnu:0.2.5
   Compiling proc-macro2 v1.0.51
   Compiling log v0.4.17
error: failed to run custom build command for `proc-macro2 v1.0.51`

Caused by:
  process didn't exit successfully: `/target/release/build/proc-macro2-7d2ae270ec6e23e9/build-script-build` (exit status: 1)
  --- stderr
  /target/release/build/proc-macro2-7d2ae270ec6e23e9/build-script-build: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.29' not found (required by /target/release/build/proc-macro2-7d2ae270ec6e23e9/build-script-build)
  /target/release/build/proc-macro2-7d2ae270ec6e23e9/build-script-build: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /target/release/build/proc-macro2-7d2ae270ec6e23e9/build-script-build)
warning: build failed, waiting for other jobs to finish...
error: failed to run custom build command for `log v0.4.17`

Caused by:
  process didn't exit successfully: `/target/release/build/log-6be2a5456205c856/build-script-build` (exit status: 1)
  --- stderr
  /target/release/build/log-6be2a5456205c856/build-script-build: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /target/release/build/log-6be2a5456205c856/build-script-build)
```

I've searched around but couldn't find any references to similar issues. Curious to know if anyone knows an obvious answer to my problem.

### What target(s) are you cross-compiling for?

x86_64-pc-windows-gnu

### Which operating system is the host (e.g computer cross is on) running?

- [X] macOS
- [ ] Windows
- [ ] Linux / BSD
- [ ] other OS (specify in description)

### What architecture is the host?

- [X] x86_64 / AMD64
- [ ] arm32
- [ ] arm64 (including Mac M1)

### What container engine is cross using?

- [X] docker
- [ ] podman
- [ ] other container engine (specify in description)

### cross version

cross 0.2.5

**Comment by Emilgardis** [comment]
this is a duplicate of #724 

Unfortunately there's not much we can do for this right now.

**Comment by bnheise** [comment]
@Emilgardis I was able to solve this issue so I want to share my solution here.

I used the tools provided in this [folder](https://github.com/cross-rs/cross-toolchains/tree/5504d9293334744cf2555286707d9907b89afbfb).

After carrying out the setup as described in the above folder, I carried out these steps:

```bash
cargo xtask configure-crosstool --glibc-version 2.29
cargo build-docker-image x86_64-pc-windows-gnu --tag local
```

Then, in my Cross.toml file:

```toml
[target.x86_64-pc-windows-gnu]
image="ghcr.io/cross-rs/x86_64-pc-windows-gnu:local"
```

After that, it worked perfectly.
